File: MusicWeb.Client/src/models/Message.js
export default class Message {
  chat_id = null;
  send_date = "";
  participant_id = null;
  participant_name = "";
  text = "";
  image_bytes = "";
  image_path = "";
  is_read = false;
  constructor($data) {
    if ($data) {
      this.chat_id = $data.chatId;
      this.send_date = $data.sendDate;
      this.participant_id = $data.senderId;
      this.participant_name = $data.senderName;
      this.text = $data.text;
      this.image_bytes = $data.imageBytes;
      this.image_path = $data.imagePath;
      this.is_read = $data.isRead;
    }
  }
}

File: MusicWeb.Server/MusicWeb.Models/Entities/Chats/Message.cs
using MusicWeb.Models.Entities.Base;
using MusicWeb.Models.Identity;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MusicWeb.Models.Entities
{
    public class Message : BaseEntity
    {
        public string Text { get; set; }
        public DateTime SendDate { get; set; }
        public string ImagePath { get; set; }

        public int ChatId { get; set; }
        public string SenderId { get; set; }
        public virtual Chat Chat { get; set; }
        public virtual ApplicationUser Sender{ get; set; }
    }
}
